src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,18 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+* Servlet class responsible for user registration.
```

Comment:
another super small thing (also not your fault) - the bottom two lines here should be moved over one space